dashboard/app/models/course_offering.rb

```diff
@@ -60,4 +66,42 @@ def self.get_from_cache(key)
       CourseOffering.find_by_key(key)
     end
   end
+
+  def serialize
+    {
+      key: key,
+      display_name: display_name
+    }.merge(properties&.sort.to_h)
```

Comment:
looks like properties are removed in https://github.com/code-dot-org/code-dot-org/pull/44586 , so I'm guessing we do not want to merge them in here